### PR TITLE
AB#74543 added land status

### PR DESCRIPTION
**Pull request by rafalgajewski**

### CODE REVIEW

File: src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+@using HE.Investment.AHP.WWW.Views.Finance.Consts
+@model Guid
+@{
+    ViewData["Title"] = "Financial details";
+}
+<div id="mainContainer" role="main">
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml:6]: why do you need mainContainer?

File: src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+<gds-div-grid-column-two-thirds>
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml:7]: I think in other places e.g. HomeTypes we use govuk-grid-column-full, are you sure about two-thirds?

File: src/HE.Investment.AHP.Contract/Domain/ValueObject.cs [hunk 3]
```diff
@@ -0,0 +1,44 @@
+namespace HE.Investment.AHP.Contract.Domain;
+
+public abstract class ValueObject
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Contract/Domain/ValueObject.cs:3]: I agree domain project is better place for that

File: src/HE.Investment.AHP.Contract/FinancialDetails/Commands/ProvideLandOwnershipAndValueCommand.cs [hunk 4]
```diff
@@ -1,6 +1,7 @@
-﻿using HE.Investment.AHP.Contract.FinancialDetails.ValueObjects;
+using HE.Investment.AHP.Contract.FinancialDetails.ValueObjects;
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Contract/FinancialDetails/Commands/ProvideLandOwnershipAndValueCommand.cs]: I've moved this to be consistent across all part of the code

File: src/HE.Investment.AHP.Contract/FinancialDetails/Models/FinancialDetailsModel.cs [hunk 5]
```diff
@@ -16,8 +16,6 @@ public FinancialDetailsModel()
 
     public string? ApplicationName { get; set; }
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investment.AHP.Contract/FinancialDetails/Models/FinancialDetailsModel.cs:16]: Agree.

File: src/HE.Investment.AHP.WWW/Controllers/FinancialDetailsController.cs [hunk 6]
```diff
@@ -25,33 +27,36 @@ public FinancialDetailsController(IMediator mediator)
...
     {
-        var result = await _mediator.Send(new StartFinancialDetailsCommand(applicationId), cancellationToken);
+        _ = await _mediator.Send(new StartFinancialDetailsCommand(applicationId, applicationName), cancellationToken);
 
-        return await Continue(new { applicationId, financialDetailsId = result.ReturnedData.FinancialDetailsId.ToString() });
+        return await Continue(new { applicationId });
     }
 
-    [HttpGet("{financialDetailsId}/land-status")]
+    [HttpGet("{applicationId}/land-status")]
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Controllers/FinancialDetailsController.cs]: Done

File: src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml (hunk 1 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml:6]: Without this I had issues with passing wcag

File: src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml (hunk 2 above)
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Views/Finance/Index.cshtml:7]: Good point. I've changed layout to full column

File: src/HE.Investment.AHP.WWW/Controllers/FinancialDetailsController.cs [hunk 7]
```diff
@@ -25,33 +27,36 @@ public FinancialDetailsController(IMediator mediator)
 
     [HttpGet("start")]
     [WorkflowState(FinancialDetailsWorkflowState.Index)]
-    public IActionResult Start(Guid applicationId)
+    public IActionResult Start(Guid applicationId, CancellationToken cancellationToken)
     {
-        return View("Index", applicationId);
+        // temporarly mocked
```
[the remark this answers is not in the document]
Comment (rafalgajewski) [reply src/HE.Investment.AHP.WWW/Controllers/FinancialDetailsController.cs:30]: Main point is to make testing of financial details independent until there will be crm added

File: src/HE.Investment.AHP.Domain/FinancialDetails/ValueObjects/ApplicationID.cs [hunk 8]
```diff
@@ -0,0 +1,27 @@
+using Dawn;
```
Comment (Antek012) [inline src/HE.Investment.AHP.Domain/FinancialDetails/ValueObjects/ApplicationID.cs:1]: fix filename

**Review by Antek012 — APPROVED** [review]

**Pull request closed without merging**